docs/assets/scss/_integration.scss

```
@@ -48,4 +48,58 @@ h6.integration > a {
+  font-weight: 600;
+  text-transform: uppercase;
+  letter-spacing: 0.03em;
+  padding: 0.2rem 0.6rem;
+  border-radius: 999px;
+  background: var(--brand-color-primary);
+  color: var(--color-white);
+}
+
+.design-card-name {
+  font-weight: 500;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)